docs/guide/setup/alchemical_network_creation.rst

```diff
@@ -0,0 +1,61 @@
+.. _alchemical_network_creation:
+
+Alchemical Networks: Creation
+=============================
+
+In :ref:`the previous section <alchemical_network_model>` we detail the
+theory of :class:`.AlchemicalNetwork` objects and what they contain. Here
+we explain how you can go about creating a :class:`.AlchemicalNetwork`
+by combining its various components.
+
+Python API
+----------
+
+You can manually create a :class:`.AlchemicalNetwork` by creating a list
+of :class:`.Transformation` objects.
+The `cookbook on creating alchemical networks <../../cookbook/create_alchemical_network.html>`_
```

Comment:
We probably shouldn't be using relative links in the docs, it's an easy way to get links to break.  You can put a `.. _tag:` in the top cell of a notebook and sphinx picks it up as an anchor

Reply:
raw tags added in https://github.com/OpenFreeEnergy/ExampleNotebooks/pull/124 and ref links added to this PR - will need to merge the ExampleNotebooks PR before it's reflected here